merge_request_approvals.go

```diff
@@ -60,6 +60,26 @@ type MergeRequestApproverGroup struct {
 	}
 }
 
+// MergeRequestApprovalRule represents a GitLab merge request approval rule.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#get-merge-request-level-rules
+type MergeRequestApprovalRule struct {
+	ID                   int                                    `json:"id,omitempty"`
+	Name                 string                                 `json:"name,omitempty"`
+	RuleType             string                                 `json:"rule_type,omitempty"`
+	EligibleApprovers    []*BasicUser                           `json:"eligible_approvers,omitempty"`
+	ApprovalsRequired    int                                    `json:"approvals_required,omitempty"`
+	SourceRule           *CreateMergeRequestApprovalRuleOptions `json:"source_rule,omitempty"`
+	Users                []*BasicUser                           `json:"users,omitempty"`
+	Groups               []*Group                               `json:"groups,omitempty"`
+	ContainsHiddenGroups bool                                   `json:"contains_hidden_groups,omitempty"`
+}
```

Comment:
`omitempty` should not be used by response structs.

Reply:
Fixed